nomos-da/full-replication/src/lib.rs

```diff
@@ -166,8 +166,8 @@ impl metadata::Metadata for Certificate {
     type AppId = [u8; 32];
     type Index = u64;
 
-    fn metadata(&self) -> Option<(Self::AppId, Self::Index)> {
-        Some((self.metadata.app_id, self.metadata.index))
+    fn metadata(&self) -> (Self::AppId, Self::Index) {
+        ([0u8; 32], 0)
```

Comment:
Why index `0` hardcoded?

Reply:
Missed this in a rebase, will revert to metadata from certificate.